sstables/sstable_set.cc

```diff
@@ -187,6 +187,29 @@ sstable_set::size() const noexcept {
     return _impl->size();
 }
 
+sstable_set sstable_set::clone_excluding(const std::vector<shared_sstable>& excluded_sstables) {
+    if (excluded_sstables.size() > size() / 2) {
+        // Typically, the number of excluded sstables is expected to be much less than the
```

Comment:
if excluded set is more than half of the whole set, the cost of inserting excluded dominates.

I think you meant less than half the size here instead?!

Reply:
oops, yes, of course.
The expression was reversed earlier and I forgot to reverse the operator as well.

Reply:
fixed in v2